agera/src/main/java/com/google/android/agera/CompiledRepository.java

```diff
@@ -49,8 +49,11 @@ static Repository compiledRepository(
       @NonNull final Merger<Object, Object, Boolean> notifyChecker,
       @RepositoryConfig final int concurrentUpdateConfig,
       @RepositoryConfig final int deactivationConfig) {
-    final Observable eventSource = perMillisecondObservable(frequency,
-        compositeObservable(eventSources.toArray(new Observable[eventSources.size()])));
+    final Observable compositeObservable =
+        compositeObservable(eventSources.toArray(new Observable[eventSources.size()]));
```

Comment:
could you make compositeObservable version that accepts `Collection<Observable>/List<Observable>`?